integration/teleterm_test.go

```diff
@@ -743,6 +749,130 @@ func testWaitForConnectMyComputerNodeJoin(t *testing.T, pack *dbhelpers.Database
 	require.NoError(t, <-waitForNodeJoinErr)
 }
 
+func testDeleteConnectMyComputerNode(t *testing.T, pack *dbhelpers.DatabasePack) {
```

Comment:
What does CMC have to do with database access? Why are we using `dbhelpers.DatabasePack` here?

Reply:
The first integration tests we added were related to db proxies ([`integration/proxy/teleterm_test.go`](https://github.com/gravitational/teleport/blob/807b9cd476d09d4891ea13ee065e5128ac60c339/integration/proxy/teleterm_test.go)). At some point, I wanted to add integration tests for other areas of lib/teleterm. It was easier for me to use `dbhelpers.DatabasePack` vs learning any other approach we use for integration tests.

Reply:
Ok, it's fine for now, but I don't want us to be writing tests that spin up database services until the end of time just because the first test we wrote worked with databases.

Reply:
I'll take a look at it. Now that I have more experience with integration tests and how Teleport works in general, it shouldn't be too hard to rewrite it to the stuff that `integration/integration_test.go` uses.

The setup of those db tests takes ages so it'll be good to get rid of it.

Reply:
I'll take a look at it the next time I add integration tests*